deliverables/android.html

```diff
@@ -1,23 +1,118 @@
+
 <!DOCTYPE html>
 <html>
 <head>
   <title>Ignition Android</title>
   <style type='text/css'>
-    div div {
-      position: relative;
-    }
+   div {margin: o; padding: 0;}
+div div {background: #2214F0; position: relative;}
```

Comment:
To help with the readability of your code (for others and for yourself in the future when you come back to some code) we try to indent our code and have only one property-value pair per line like so:

``` css
<style type='text/css'>
  div {
    margin: 0; 
    padding: 0;
  }

  div div {
    background: #2214F0; 
    position: relative;
  }

  .android {
    height: 404px; 
    width: 334px;
    margin: 100px auto;
  }
...
```